spec/support/factory_bot.rb

```diff
@@ -1,5 +1,5 @@
-# # spec/support/factory_bot.rb
+# spec/support/factory_bot.rb
 
-# RSpec.configure do |config|
-#   config.include FactoryBot::Syntax::Methods
-# end
+RSpec.configure do |config|
+  config.include FactoryBot::Syntax::Methods
+end
```

Comment:
![Capture](https://user-images.githubusercontent.com/63087491/215932075-5f2c36fd-e634-4142-9cf2-dde81a6db113.PNG)


- [ ] I'm unable to set up your app locally through `rails db:setup` as a result of the error above. Kindly add the line ` require 'rspec/rails'`  or `require 'rspec/core'` at the top of spec/support/factory_bot.rb  file

Reply:
Change done as per your instruction